CHANGES.rst

```diff
@@ -121,6 +121,10 @@ astropy.table
 
 - Added a new ``showtable`` command-line script to view binary or ASCII table
   files. [#6859]
+- Added new table property ``astropy.table.loc_indices`` that returns the
+  location of rows by indexes. [#6831]
+
+- Allow updating of table by indices through the property ``astropy.table.loc``. [#6831]
```

Comment:
These are attributes of the `Table` class, so should be `Table.loc` and `Table.loc_indices` (or `astropy.table.Table.loc` if you want the full name, I think it's fine with the short name). And a newline is missing before the first item.